utils/src/logging.rs

```diff
@@ -9,8 +13,8 @@ pub fn setup_logging() -> Result<()> {
 
 #[cfg(not(feature = "tracing-forest"))]
 pub fn subscriber() -> impl Subscriber + core::fmt::Debug {
-    use tracing::level_filters::LevelFilter;
-    use tracing_subscriber::{fmt::format::FmtSpan, EnvFilter};
+    println!("using default");
```

Comment:
If this line added not only for testing, we need to have more informative message here.

Reply:
oops, it was for testing only, removed it :)